toponetx/transform/graph_to_simplicial_complex.py

```diff
@@ -60,3 +62,40 @@ def graph_2_clique_complex(G, max_dim=None):
 
     lst = filter(lambda face: len(face) <= max_dim, nx.enumerate_all_cliques(G))
     return SimplicialComplex(list(lst))
+
+
+def weighted_graph_to_Vietoris_Rips_complex(G, r, max_dim=None):
+    """Get the Vietoris-Rips complex of radius r of a weighted undirected graph. The Vietoris-Rips complex of radius
+    r is the clique complex given by the cliques of G whose nodes have pairwise distances less or equal than r. All
+    vertices are added to the Vietoris-Rips complex regardless the radius introduced. If G is a clique weighted by a
+    dissimilarity function d that satisfies max_v d(v, v) <= min d(u,v) for u != v, and r >= d(v, v) for all nodes v,
+    then the Vietoris-Rips complex of radius r is the usual Vietoris-Rips abstract simplicial complex of radius r for
+    point clouds with dissimilarities.
+
+    Parameters
+    ----------
+    G : networkx graph
+        Weighted undirected input graph. The weights of the edges must be in the attribute 'weight'.
+    r : float
+        The radius for the Vietoris-Rips simplicial complex computation.
+    max_dim : int, optional
+        The max dimension of the cliques in
+        the output clique complex.
+        The default is None indicate max dimension.
+
+    Returns
+    -------
+    SimplicialComplex
+        The Vietoris-Rips simplicial complex of dimension max_dim of the graph G.
+        """
+
+    def is_in_VR_complex(clique):
+        # Check if all edge weights are less or equal than r
```

Comment:
No unnecessary comments, these are hard to maintain. Put this in a docstring.

Reply:
this has not been addressed

Reply:
Hi @mhajij! I removed this comment in the latest commit, but I forgot to answer Nina. Is it right now?